src/components/Table/Table.tsx

```diff
@@ -52,39 +45,22 @@ interface Props {
    * If loading failed....
    */
   loadingFailed?: boolean;
-  /**
-   * Callback function to execute when a sortable column's header is clicked.
-   */
-  onSort?: (event: ChangeEvent<HTMLInputElement>) => void;
-  /**
-   * The current sorted column.
-   */
-  sortedColumn?: SortedColumn;
-  /**
-   * If content overruns its cell's width, truncate the content, as opposed to wrapping it.
-   */
-  truncateOverflow?: boolean;
   /**
    * Fix the width of the columns. Can be useful if sorting is enabled and the content of
    * the columns is changing; prevents the horizontal jump when this occurres.
    */
   useFixedWidthColumns?: boolean;
 }
 
-const Table: FC<Props> = ({
-  columns,
-  tableData,
+const Table: FC<TableProps> = ({
   className = undefined,
-  emptyCellPlaceholder = undefined,
+  children = null,
```

Comment:
Wouldn't children be required here? In this case, children referrers to TableBody, and only TableBody; there should never be a Table without a TableBody.

In this case, the <tbody> tag should probably be included in Table, to prevent someone from doing something foolish like forgetting it. 

Otherwise, we need a way to validate that the child is in fact the TableBody component.

Reply:
Not really since `TableHead` is outside of `TableBody` the table can take both as separate nodes. Not sure how to best describe this but it should be flexible that way.